closed/autoconf/custom-spec.gmk.in

```diff
@@ -130,8 +130,10 @@ ifeq ($(OPENJDK_BUILD_OS), macosx)
   # is not applied.
   export MACOSX_DEPLOYMENT_TARGET := @MACOSX_VERSION_MIN@
 
-  # Set page zero size to 4KB for mapping memory below 4GB.
-  LDFLAGS_JDKEXE += -pagezero_size 0x1000
+  ifneq ($(OPENJDK_TARGET_CPU), aarch64)
```

Comment:
I suspect this is the reason why the 'VM fails to start in the compressed refs mode' as noted in https://github.com/eclipse-openj9/openj9/issues/11164#issuecomment-943101232; that's the reason why it was added for x86_64. What happens if `-pagezero_size 0x1000` is used on aarch64?

Reply:
A quick read of https://developer.apple.com/forums/thread/655950 and https://developer.apple.com/documentation/security/preparing_your_app_to_work_with_pointer_authentication suggests that perhaps we won't be able to support compressed references on aarch64 (at least not how it's currently done).

Reply:
To be clear, shifting (alone) won't help, we need to add an offset (to the start of that allocated block).

Reply:
We don't need to point to the start of the block, just inside it. We can allocate more (virtual) memory than needed so the offset isn't required. Only the pages touched in the virtual block will use physical memory.